app/src/main/res/layout/settings.xml

```diff
@@ -156,11 +156,29 @@
                         <org.mozilla.vrbrowser.ui.views.HoneycombButton
                             android:id="@+id/controllerOptionsButton"
                             style="?attr/honeycombButtonStyle"
+                            android:layout_alignTop="@+id/second_row"
```

Comment:
This should be `android:visibility="gone"` instead?

Reply:
yeah, must've changed it visually in the View